src/components/TradeAssetSearch/TradeAssetSearch.tsx

```diff
@@ -36,7 +36,7 @@ const assetButtonProps = {
   pr: 3,
   py: 1.5,
   size: 'sm',
-  borderRadius: 'full',
+  borderRadius: '2xl',
```

Comment:
it looks visually off if there is both a large x-margin and a full boarder radius.